dotnet-roslyn/config/Audacia.CodeAnalysis/Audacia.CodeAnalysis.csproj

```diff
@@ -12,11 +12,17 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Audacia.CodeAnalysis.Analyzers" Version="1.11.0" />
+    <PackageReference Include="Audacia.CodeAnalysis.Analyzers" Version="1.11.2" />
     <PackageReference Include="CSharpGuidelinesAnalyzer" Version="3.8.5" />
     <PackageReference Include="DotNetAnalyzers.DocumentationAnalyzers" Version="1.0.0-beta.59" />
-    <PackageReference Include="IDisposableAnalyzers" Version="4.0.7" />
-    <PackageReference Include="Roslynator.Analyzers" Version="4.12.3" />
+    <PackageReference Include="IDisposableAnalyzers" Version="4.0.8">
+      <PrivateAssets>all</PrivateAssets>
```

Comment:
I don't think you want the `PrivateAssets` and `IncludeAssets` in here, means these packages won't be installed by consumers of Audacia.CodeAnalysis

Reply:
Happy to remove it but that's how it's done with some of the other .csproj files e.g
![image](https://github.com/user-attachments/assets/bfb0d71d-b000-44b3-acbe-fba3fe762e01)